addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthTestDialog.java

```diff
@@ -455,12 +460,45 @@ public String getSaveButtonText() {
         return Constant.messages.getString("authhelper.auth.test.dialog.button.save");
     }
 
+    @Override
+    public JButton[] getExtraButtons() {
+        if (extraButtons == null) {
+            JButton resetButton =
+                    new JButton(
+                            Constant.messages.getString(
+                                    "authhelper.auth.test.dialog.button.reset"));
+            resetButton.addActionListener(
+                    e -> {
+                        ((JTextField) getField(CONTEXT_LABEL))
```

Comment:
I'd have expected the usage of `setFieldValue` instead of doing casts, any reason to do this way?

Reply:
I just didn't know any better. Will adjust.

Reply:
Adjusted

Reply:
And tested live.

Reply:
The browser is not being reset, on purpose?

Reply:
I was waffling about it. But you're right there's no reason not to.

Reply:
Done

Reply:
Might not be done properly, testing/checking

Reply:
Sorted, thanks!